libs/vm-modules/src/ml/utilities/factory_with_estimator.cpp

```diff
@@ -0,0 +1,89 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "vm_modules/math/utilities/factory_with_estimator.hpp"
+#include "vm_modules/ml/utilities/factory_with_estimator.hpp"
+
+namespace vm_modules {
+namespace ml {
+namespace factory_with_estimator {
+
+namespace maf = math::factory_with_estimator;
+
+//////////////////////////////
+// VM model related helpers //
+//////////////////////////////
+
+fetch::vm::Ptr<fetch::vm_modules::ml::model::VMModel> vmSequentialModel(VMPtr &vm)
+{
+  auto model_category = maf::vmString(vm, "sequential");
+  auto model          = vm->CreateNewObject<fetch::vm_modules::ml::model::VMModel>(model_category);
+  return model;
+}
+
+fetch::vm::Ptr<fetch::vm_modules::ml::model::VMModel> vmSequentialModel(
```

Comment:
i'm not hugely convinced by these factory functions. They're going to get a lot more messy and complicated when we enable multiple activations and multiple layer types, rather than all just dense/relu